### ISSUE TITLE
VocabTree matching doesn't work

### ISSUE THREAD

**Issue by kuynzereb** [colmap/pycolmap#149]
I am trying to use VocabTree matching and got the following exception
```
==============================================================================
Vocabulary tree feature matching
==============================================================================

terminate called after throwing an instance of 'flann::FLANNException' 
  what():  Invalid index file, wrong signature 
```
The matching is called like that 
```
opts = pycolmap.VocabTreeMatchingOptions()
opts.num_images = 30
opts.vocab_tree_path = "/mnt/hdd/vocab_tree_flickr100K_words32K.bin"
pycolmap.match_vocabtree(database_path, matching_options=opts)
```
Is it a bug or maybe I am doing something wrong?

**Comment by sarlinpe** [comment]
I cannot reproduce this at HEAD. There was a bug in v0.3, which has been fixed for v0.4 by https://github.com/colmap/pycolmap/pull/128.
```python
import pycolmap
image_dir = '...'
database_path = '...'
pycolmap.extract_features(database_path, image_dir)

vocab_path = 'vocab_tree_flickr100K_words32K.bin'
# vocab_path = 'vocab_tree_flickr100K_words256K.bin'
# vocab_path = 'vocab_tree_flickr100K_words1M.bin'
!wget https://demuc.de/colmap/$vocab_path -O $vocab_path

opts = pycolmap.VocabTreeMatchingOptions(vocab_tree_path=vocab_path)
opts.check()
pycolmap.match_vocabtree(database_path, matching_options=opts)
```

**Comment by kuynzereb** [comment]
Thank you very much!

I don't understand why it didn't work for me previously, but now it indeed works.
